src/features/GenerateBugReport.ts

```diff
@@ -0,0 +1,133 @@
+import vscode = require('vscode');
+import { SessionManager } from '../session';
+import cp = require('child_process');
+import Settings = require('../settings');
+
+import window = vscode.window;
+const os = require("os");
+
+import { IFeature, LanguageClient } from '../feature';
+// import { IExtensionManagementService, LocalExtensionType, ILocalExtension } from 'vs/platform/extensionManagement/common/extensionManagement';
+
+const extensionId: string = 'ms-vscode.PowerShell';
+const extensionVersion: string = vscode.extensions.getExtension(extensionId).packageJSON.version;
+
+const queryStringPrefix: string = '?'
+
+var settings = Settings.load();
+let project = settings.bugReporting.project;
+
+const issuesUrl: string = `${project}/issues/new`
+
+var extensions = vscode.extensions.all.filter(element => element.packageJSON.isBuiltin == false).sort((leftside, rightside): number => {
+    if (leftside.packageJSON.name.toLowerCase() < rightside.packageJSON.name.toLowerCase()) return -1;
+    if (leftside.packageJSON.name.toLowerCase() > rightside.packageJSON.name.toLowerCase()) return 1;
+    return 0;
+})
+
+export class GenerateBugReportFeature implements IFeature {
+
+    private command: vscode.Disposable;
+    private powerShellProcess: cp.ChildProcess;
+
+    constructor(private sessionManager: SessionManager) {
+        this.command = vscode.commands.registerCommand('PowerShell.GenerateBugReport', () => {
+
+
+            var OutputChannel = window.createOutputChannel('Debug');
```

Comment:
No, actually that was just used for debugging my code. I will remove it. Unless you think it would be beneficial to put the information into the log file.